client/src/Clusters/RecommendedCategories/RecommendedCategories.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
```

Comment:
I would use snake case for file names. I.e. recommended_categories.js

Same with the css file below